test/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/NRLMSISE00.test.cpp

```diff
@@ -306,12 +311,10 @@ TEST(OpenSpaceToolkit_Physics_Environment_Atmospheric_Earth_NRLMSISE00, GetDensi
 
             Instant instant = Instant::DateTime(datetime, Scale::UTC);
 
-            const Position sunPosition = sunFrameSPtr->getOriginIn(Frame::ITRF(), instant);
-
-            const Real density = nrlmsise.getDensityAt(position, instant, sunPosition);
+            const Real density = nrlmsise.getDensityAt(lla, instant);
 
             // Check percent tolerance here for low altitudes
-            const Real percentTolerance = 0.5;
+            const Real percentTolerance = 0.42;
```

Comment:
Is this the expected tolerance? Seems a bit high no?

Reply:
Yeah this is the best tolerance I could get for these tests.

Reply:
Hmmmm okay. Maybe worth discussing this in person I'm a little confused.

Reply:
Yeah down to chat more about it

Reply:
Oh wait it's 0.42%. Not 42%. 😮‍💨